Src/library/net/Clib/network.c

```diff
@@ -775,11 +778,108 @@ EIF_INTEGER c_put_stream_noexception(EIF_INTEGER fd, EIF_POINTER stream_pointer,
 }
 
 void c_put_stream(EIF_INTEGER fd, EIF_POINTER stream_pointer, EIF_INTEGER length)
-	/*x transmission of string s of size size trought socket fd */
+	/*x transmission of string s of size length throught socket fd */
 {
 	eif_net_check(c_put_stream_noexception(fd, stream_pointer, length));
 }
 
+EIF_INTEGER c_unoptimized_sendfile_noexception(EIF_INTEGER out_fd, FILE* f, EIF_INTEGER length)
```

Comment:
I did not see the unoptimized one being used on my first review, so I thought the choice was made at the Eiffel level. I would call it `c_sendfile` and `c_sendfile_fallback`. No need for the `noexception` suffix because this is a a new API and we cannot add that suffix each time.